fem/tmop.hpp

```diff
@@ -1523,6 +1638,11 @@ class TMOP_Integrator : public NonlinearFormIntegrator
    void AssemblePA_Limiting();
    void ComputeAllElementTargets(const Vector &xe = Vector()) const;
 
+   double ComputeUntanglerMinDetT(const Vector &x,
+                                  const FiniteElementSpace &fes);
+   double ComputeUntanglerMaxMuT(const Vector &x,
```

Comment:
I thought this was not necessarily untangle-related. How about `ComputeMaxMetricValue`?